app/services/users/profile.rb

```diff
@@ -135,14 +135,24 @@ def vet360_contact_information
         fax_number: person.fax_number
       }
     rescue => e
-      error_hash = Users::ExceptionHandler.new(e, 'VAProfile').serialize_error
-      scaffold.errors << error_hash
-      log_external_service_error(error_hash)
+      handle_service_error(e, 'VAProfile', 'vet360_contact_information')
       nil
     end
 
     # rubocop:disable Metrics/MethodLength
     def mpi_profile
+      unless user.loa3?
```

Comment:
This change addresses the 401 Unauthorized error we were getting from MPI. Previously, this resulted in a 296 status, but now we handle it gracefully by returning nil and logging the error for visibility.